packages/cloud-cognitive/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -776,14 +776,22 @@ export const LeftPanel = () => {
   const columns = React.useMemo(() => defaultHeader, []);
   const [data] = useState(makeData(10));
   const datagridState = useDatagrid({
+    leftPanel: {
+      isOpen: true, // this toggling will happen from datagridActions.
+      panelContent: (
+        <div style={{ display: 'inline' }}>
```

Comment:
@faraznaseem019 

Maybe let's move this to a `*.scss` file.

While this works, inline styling is more commonly used when style is dynamically updated based on a state variable. For example, we may want to adjust the `background-color `based on minute changes in screen size - here probably want to declare the selector inline as we could end up with an infinite number of classes to cater for the possible RGB values.

I do note this is for a story, and if it's consistent with how styles are being applied for stories it could be okay to leave as is.


cc: @holmansze @matthewgallo